js/app/modules/arrangement/sideBySide.js

```diff
@@ -51,66 +72,73 @@ const _SideBySideLayout = new Knowledge.Class({
         if (all_cards.length === 0)
             return [0, 0];
 
-        let [min, nat] = all_cards[0].get_preferred_width();
+        let [, first_nat] = all_cards[0].get_preferred_width();
+        let [, button_nat] = this._popover_button.get_preferred_width();
+
+        let min = first_nat + SPACING_MIN + button_nat;
+        let nat = 0;
 
-        nat += all_cards.slice(1).reduce((accum, card) => {
+        all_cards.forEach ((card) => {
```

Comment:
Aw, I liked the `reduce()` loop :stuck_out_tongue_closed_eyes:

nitpick: delete space before `(`